applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -130,9 +130,12 @@ namespace Kratos
 		const double A = this->GetProperties()[CROSS_AREA];
 		const double L = this->CalculateReferenceLength();
 
-		const double J = this->GetProperties()[IT];
-		const double Iy = this->GetProperties()[IY];
-		const double Iz = this->GetProperties()[IZ];
+		Matrix inertia = GetProperties()[LOCAL_INERTIA_TENSOR];
```

Comment:
alright I'll change it to vector, it will also be more convinient to use, thanks.